README.md

```diff
@@ -4,23 +4,25 @@ A demo project exploring tagless-final DSL implementation in OCaml with Python b
 
 ## Overview
 
-`pyfinalo` is an experimental domain-specific language (DSL) implementation that showcases the feasibility to implement powerful eDSL usable in Python backed by implementation in OCaml in the tagless-final style. The project provides multiple interpreters including direct evaluation, AST construction, and explanatory type checking.
+`pyfinalo` is an experimental domain-specific language (DSL) implementation that showcases the feasibility to implement powerful eDSL usable in Python and JavaScript/TypeScript backed by implementation in OCaml in the tagless-final style. The project provides multiple interpreters including direct evaluation, AST construction, and explanatory type checking.
```

Comment:
add a paragraph on the naming:

this proj was originally supposed to be a demo for python edsl impl with ocaml in tagless-final style. js support was added later completely as an accident.

therefore call out that the project name now is a misnomer (and maybe make some fun comments about it if you see fit)